recipe/meta.yaml

```
@@ -1,42 +1,60 @@
 {% set name = "inequality" %}
-{% set version = "1.0.0" %}
+{% set version = "1.1.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

```suggestion
  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
I recently learned that `.org` is preferred TLD for recipes. It also appears in CF feedstock

---

prefer `.io` URL for pypi packages